elements/lisk-transactions/src/0_transfer_transaction.ts

```diff
@@ -260,4 +260,16 @@ export class TransferTransaction extends BaseTransaction {
 
 		return errors;
 	}
+
+	// tslint:disable:next-line: prefer-function-over-method no-any
+	protected assetDbRead(raw: any): object | undefined {
```

Comment:
@shuse2 Do we have `db` in the `2.0`? if not then why we have it in transaction? 


So these are the methods with others have to implement in their custom transactions. So we should straight up the naming convention right now.

Reply:
It's the implementation for backward compatibility, and just moving the function to the transaction
it will be removed on next hardfork

Reply:
These are the methods with others have to implement in their custom transactions?

Reply:
BTW We renamed the db to storage a long time earlier. Why we even followed this naming convention from first place.

Reply:
this is not even `storage` convention.
It's legacy left from `db`, and the `storage` is not related
and yes user have to implement it, otherwise sync process doesn't work well ='(
correct? @mitsujutsu

Reply:
If user have to implement `assetDbRead` method in their custom transaction after 2.0 release. Then we must fix this naming convention right now.

Reply:
That's correct user will have to implement it, but only until the hardfork. This is necessary as a work around to allow for custom tx in the meantime.